.kokoro/presubmit/downstream-build.sh

```diff
@@ -15,6 +15,8 @@
 
 set -eo pipefail
 
+SHOWCASE_VERSION=0.26.1
```

Comment:
We may want to update [renovate.json](https://github.com/googleapis/gapic-generator-java/blob/main/renovate.json#L30-L39) to update this version automatically.